forc/src/cli/commands/new.rs

```diff
@@ -43,6 +44,20 @@ pub(crate) fn exec(command: Command) -> Result<()> {
         path,
     } = command;
 
+    match &name {
+        Some(name) => validate_name(name, "project name")?,
+        None => {
+            // If there is no name specified for the project directory name will be used by default
```

Comment:
```suggestion
            // If there is no name specified for the project, the 'path' input will be used by default
```

minor docs nit here

Reply:
I explicitly stated directory name as the name of the created dir is what we need to check not the path (that is why we are getting the 'file_name' from the path before checking.

Reply:
Ah, got it. I think maybe I got a bit confused myself reading it😅